listeners.js

```diff
@@ -192,5 +192,29 @@ module.exports = function startListeners(ssb, onError) {
         }
       )
     )
+
+    // re-add missing people to a new epoch if the epoch creator didn't
+    // we're only doing this for the preferred epoch atm
+    pull(
+      ssb.db.query(
+        where(type('group/init')),
+        live({ old: true }),
+        toPullStream()
+      ),
+      pull.filter((msg) => isInitEpoch(msg)),
+      pull.drain(
+        (msg) => {
+          // TODO: add random timeout
```

Comment:
@mixmix note that I think both you and I want to add "slight random timeout" around now. i think you might get to it first but in that case put it somewhere i can easily reuse maybe? :)

Reply:
for 4.6 i think.

could also be nice with a config that lets us put it to 0 in tests

Reply:
did this here now https://github.com/ssbc/ssb-tribes2/blob/bdf1a2ee676c8ff27a99a5c22bd156f07a51d4ff/listeners.js#L211-L222

should be easily extractable to a helper function

Reply:
extracting this in https://github.com/ssbc/ssb-tribes2/pull/127